### PR TITLE
Grammmar fix

### PR DESCRIPTION
**Pull request by jakmak3**
This change fixes a minor grammar error in the cyborg jetpack module. The name of the module implies it is only used for mining cyborgs when in fact is used in mining borgs and engineering borgs.

### CODE REVIEW

**Comment by The-Albinobigfoot** [comment]
This isn't grammar - it's semantics.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -168,8 +168,8 @@
 		T.recharge_time = max(2 , T.recharge_time - 4)
 
 /obj/item/borg/upgrade/jetpack
-	name = "mining robot jetpack"
-	desc = "A carbon dioxide jetpack suitable for low-gravity mining operations."
+	name = "utility robot jetpack"
+	desc = "A carbon dioxide jetpack suitable for low-gravity mining and engineering operations."
```
Comment (The-Albinobigfoot) [inline code/game/objects/items/robot/robot_upgrades.dm]: > A carbon dioxide jetpack suitable for low-gravity operations.

Reads better tbh.

**Review by The-Albinobigfoot — CHANGES_REQUESTED** [review]

File: code/game/objects/items/robot/robot_upgrades.dm (hunk 1 above)
Comment (jakmak3) [reply code/game/objects/items/robot/robot_upgrades.dm]: so remove the mining and engineering from the entire sentance?

Comment (The-Albinobigfoot) [reply code/game/objects/items/robot/robot_upgrades.dm]: sounds good to me

**Review by The-Albinobigfoot — APPROVED** [review]
:+1:

**Comment by TheFriggenSawbones** [comment]
good luck

**Pull request merged**